server.go

```diff
@@ -22,8 +22,23 @@ const (
 		"`Authorization` header with any valid looking testmode secret API " +
 		"key. For example, `Authorization: Bearer sk_test_123`. " +
 		"Authorization was '%s'."
+
+	invalidRoute = "Unrecognized request URL (%s: %s)."
+
+	internalServerError = "An internal error occurred."
+
+	typeInvalidRequestError = "invalid_request_error"
 )
 
+type ErrorInfo struct {
+	Type    string `json:"type"`
+	Message string `json:"message"`
+}
+
+type ResponseError struct {
+	ErrorInfo ErrorInfo `json:"error"`
```

Comment:
Given that `ErrorInfo` never needs to be addressed directly, this is probably a good place to use an anonymous struct like:

``` go
type ResponseError struct {
    ErrorInfo struct {
        ...
    } `json:"error"`
}
```

Not too big of a deal though.

Reply:
@brandur-stripe okay so I fixed this one but I had to repeat the struct definition in [createStripeError()](https://github.com/stripe/stripe-mock/pull/32/commits/23e9582c430ebe0bac4248252043a1c2a5136a38#diff-34c6b408d72845d076d47126c29948d1R441) which feels potentially less clean (though maybe more idiomatic)

Reply:
Yeah, that's an unfortunate side effect. That said, I think it works well as you have it now. Thanks!